src/heart.c

```diff
@@ -158,6 +159,7 @@ static int  wait_until_close_write_or_env_tmo(int);
 /*  static variables */
 
 static const char *watchdog_path = "/dev/watchdog0";
+static const char *custom_error_log = "/var/system/nerves_heart_error.log";
```

Comment:
instead of hardcoding it we could provide a command line argiment like -log see get_arguments function

Reply:
This is what I wanted to do, but that is not as trivial as it looks. If you look at that get_arguments functions, it's not clear to me how the arguments he parses for match up to the ones you have to give it in the README. He parse for '-h' and '-p', yet in the README, he mentions only how to add a HEART_BEAT_TIMEOUT.

Reply:
yup wonders of ye open-source :D documentatio is ebing put on a long finger